RedHatStyleGuide/en-US/Release_05.xml

```diff
@@ -9,28 +9,31 @@
 		Each chapter is a major release.
 	</para>
 	 <section id="sect-Red_Hat_Technical_Writing_Style_Guide-Release_Notes-Release_5-Release_5.1">
-		<title>Release 5.1</title>
-		 <para>
-			This is a test paragraph in a section
-		</para>
+		<title>Release 5.1, January 2022</title>
```

Comment:
Not a huge fan of the date in the title. Can we have an "intro" para before the list of updates that says something along the lines of "release X was release on `date` and includes the following updates:" or something similar?